[PATCH] ARM: dts: imx6qdl-udoo: Disable usbh1 to avoid kernel hang

Currently the kernel hangs when USB Host1 is enabled due to the lack of
support for controlling the USB hub clock and GPIO reset line.

Peter Chen has made several attempts to fix this problem, but his series
has not been applied yet, so better disable USB host1 for now to avoid
the kernel hang.

Signed-off-by: Fabio Estevam <fabio.estevam@nxp.com>
---

diff --git a/arch/arm/boot/dts/imx6qdl-udoo.dtsi b/arch/arm/boot/dts/imx6qdl-udoo.dtsi
index 828dd20cd27d..c9e122865312 100644
--- a/arch/arm/boot/dts/imx6qdl-udoo.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-udoo.dtsi
@@ -292,7 +292,8 @@ &usbh1 {
 	pinctrl-0 = <&pinctrl_usbh>;
 	vbus-supply = <&reg_usb_h1_vbus>;
 	clocks = <&clks IMX6QDL_CLK_CKO>;
-	status = "okay";
+	/* currently USB support causes a kernel hang. Disable it for now */
+	status = "disabled";
 };
 
 &usbotg {
